ui/raidboss/data/07-dt/ultimate/futures_rewritten.ts

```diff
@@ -217,14 +223,16 @@ const triggerSet: TriggerSet<Data> = {
   comments: {
     en: 'Triggers: P1-3 / Timeline: P1-5',
     cn: '触发器: P1-3 / 时间轴: P1-5',
+    ko: '트리거: P1-3 / 타임라인: P1-5',
   },
   config: [
     {
       id: 'sinboundRotate',
       comment: {
         en:
           `Always Away, Cursed Clockwise: <a href="https://pastebin.com/ue7w9jJH" target="_blank">LesBin<a>`,
-        cn: `总是远离,·180°·顺时针:·<a·href="https://pastebin.com/ue7w9jJH"·target="_blank">LesBin<a>`,
+        cn: `总是远离,·180°·顺时针:·<a href="https://pastebin.com/ue7w9jJH" target="_blank">LesBin<a>`,
+        ko: `항상 멀리, 180도 시계방향: <a href="https://pastebin.com/ue7w9jJH" target="_blank">LesBin<a>`,
```

Comment:
```suggestion
        cn: `总是远离,·180°·顺时针:·<a href="https://pastebin.com/ue7w9jJH" target="_blank">LesBin</a>`,
        ko: `항상 멀리, 180도 시계방향: <a href="https://pastebin.com/ue7w9jJH" target="_blank">LesBin</a>`,
```
All of the links in this file are missing proper closing tags 🤦

Reply:
I copied and pasted without much thought, but I think it would be better to fix it for all languages.